### ISSUE TITLE
Crash on Windows

### ISSUE THREAD

**Issue by caxanga334** [TF2-DMB/CBaseNPC#34]
# Basic Information

* SM Version: 1.12.0.6943
* Metamod Version: 1.12.0-dev+1164
* CBaseNPC Version: 1.8.1.96

# Description

I'm using this extension to develop a bot for TF2. After the VScript update, I'm getting crashes on **Windows** only.

I'm not sure what exactly is causing the crash. I know it happens when the bot enters my "Roam" action. It might be something related to PathFollower, ILocomotion or NavMesh.
[Link to crash dump.](https://crash.limetech.org/2pssi5djnlwv)

On my Linux server, everything is working as expected.

**Comment by caxanga334** [comment]
The crash also happens with the [scout nextbot example plugin](https://github.com/TF2-DMB/CBaseNPC/blob/master/scripting/cbasenpc_actiontest.sp).

## Plugin List

```
[SM] Listing 18 plugins:
  01 "Admin File Reader" (1.12.0.6942) by AlliedModders LLC
  02 "Admin Help" (1.12.0.6942) by AlliedModders LLC
  03 "Admin Menu" (1.12.0.6942) by AlliedModders LLC
  04 "Anti-Flood" (1.12.0.6942) by AlliedModders LLC
  05 "Basic Ban Commands" (1.12.0.6942) by AlliedModders LLC
  06 "Basic Chat" (1.12.0.6942) by AlliedModders LLC
  07 "Basic Comm Control" (1.12.0.6942) by AlliedModders LLC
  08 "Basic Commands" (1.12.0.6942) by AlliedModders LLC
  09 "Basic Info Triggers" (1.12.0.6942) by AlliedModders LLC
  10 "Basic Votes" (1.12.0.6942) by AlliedModders LLC
  11 "cbasenpc_actiontest.smx"
  12 "Client Preferences" (1.12.0.6942) by AlliedModders LLC
  13 "Fun Commands" (1.12.0.6942) by AlliedModders LLC
  14 "Fun Votes" (1.12.0.6942) by AlliedModders LLC
  15 "Nextmap" (1.12.0.6942) by AlliedModders LLC
  16 "Player Commands" (1.12.0.6942) by AlliedModders LLC
  17 "Reserved Slots" (1.12.0.6942) by AlliedModders LLC
  18 "Sound Commands" (1.12.0.6942) by AlliedModders LLC
```

## Extension List

```
[SM] Displaying 10 extensions:
[01] Accelerator (2.5.0): SRCDS Crash Handler
[02] Webternet (1.12.0.6943): Extension for interacting with URLs
[03] TF2 Tools (1.12.0.6943): TF2 extended functionality
[04] BinTools (1.12.0.6943): Low-level C/C++ Calling API
[05] SDK Hooks (1.12.0.6943): Source SDK Hooks
[06] SDK Tools (1.12.0.6943): Source SDK Tools
[07] Top Menus (1.12.0.6943): Creates sorted nested menus
[08] CBaseNPC (1.8.1.96): Primarily exposes valve's NextBot framework. But also various functions from public SDK.
[09] Client Preferences (1.12.0.6943): Saves client preference settings
[10] SQLite (1.12.0.6943): SQLite Driver
```

**Comment by KitRifty** [comment]
Thanks for the report! This should be fixed by #35.

**Comment by caxanga334** [comment]
Confirming this is fixed by #35.

**Comment by fearts** [comment]
Seems to be fixed for me as well. Big thank you to both of you for pushing this update out so quickly. I appreciate you.
